tests/unit_node/http_test.ts

```diff
@@ -1796,10 +1805,6 @@ Deno.test("[node/http] upgraded socket closes when the server closed without clo
 
       console.log("process closed");
       serverProcessClosed.resolve();
-
-      // sending some additional message
-      socket.write(Buffer.from("81847de88e01", "hex"));
-      socket.write(Buffer.from("0d81e066", "hex"));
```

Comment:
Why is this removed?

Reply:
these cause errors because the stream is closed. the duplex was hiding this.